src/App.js

```diff
@@ -1,25 +1,145 @@
-import logo from './logo.svg';
-import './App.css';
+import Button from "./components/Button";
+import { useState } from 'react';
+
+
+const App = () => {
+  const values = ["AC", "+/-", "%", "/", "7", "8", "9", "*", "4", "5", "6", "-", "1", "2", "3", "+", "0", ".", "="]
+
+  const [displayValue, setDisplayValue] = useState("")
+
+  const [operation, setOperation] = useState({ operand1: "", operator: null, operand2: "" })
+
+  const [results, setResults] = useState("")
+
+  const selectOperator = (operator) => {
+    setOperation(prevOperation => ({ ...prevOperation, operator: operator }))
+    setDisplayValue("")
+  }
+
+  const selectOperand1 = (operand1) => { setOperation(prevOperation => ({ ...prevOperation, operand1: prevOperation.operand1 + operand1 })) }
```

Comment:
In all instances, including this one, where arrow functions have only one statement, the curly braces can be omitted. So, for example, the function "selectOperand1" can be rewritten without the curly braces as follows:
```suggestion
  const selectOperand1 = (operand1) => setOperation(prevOperation => ({ ...prevOperation, operand1: prevOperation.operand1 + operand1 })) 
```